kratos/containers/pointer_vector_set.h

```diff
@@ -543,75 +529,125 @@ class PointerVectorSet final
      */
     void push_back(TPointerType x)
```

Comment:
i thought we wanted to remove this method

Reply:
This will be deprecated pointing to the `insert`. In this PR, I will only replace push_back uses in the core only. In another PR, I will change the application uses of `push_back`.